react-bootstrap/package.json

```diff
@@ -26,7 +26,8 @@
   "homepage": "https://www.runly.io/docs/integration/js/react-bootstrap/",
   "dependencies": {
     "@runly/core": "^0.1.3",
-    "lodash": "^4.17.15"
+    "lodash": "^4.17.15",
+    "react-helmet": "^5.2.1"
```

Comment:
Should we make this a peer dependency?

Reply:
We'll have to make it a dependency in the html bundle then, correct?

Reply:
yeah

Reply:
ehh on second thought, let's leave this like it is